src/browser/components/Tab/Tab.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import classNames from 'classnames';
+import { bool, string, func } from 'prop-types';
+import { noop } from 'lodash';
+
+import Icon from 'shared/components/Icon';
+
+import styles from './Tab.scss';
+
+export default class Tab extends React.Component {
+  static propTypes = {
+    className: string,
+    active: bool,
+    title: string.isRequired,
+    loading: bool,
+    onClick: func,
+    onClose: func
+  };
+
+  static defaultProps = {
+    className: null,
+    active: false,
+    loading: false,
+    onClick: noop,
+    onClose: noop
+  };
+
+  render() {
+    const { className, active, title, onClick } = this.props;
+
+    return (
+      <div
+        className={classNames(className, styles.tab, { [styles.active]: active })}
+        role="button"
+        tabIndex={0}
```

Comment:
why is this 0?

Reply:
> `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, but its order is defined by the document's source order.

[Source](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)

Basically, any element that has a click handler but doesn't natively support being clicked by the browser should have a role and a tabIndex.